cluster/examples/kubernetes/ceph/csi/example/cephfs/storageclass.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-cephfs
+provisioner: csi-cephfsplugin
+parameters:
+  # Comma separated list of Ceph monitors
+  # if using FQDN, make sure csi plugin's dns policy is appropriate.  
+  monitors: mon1:port,mon2:port,...
+
+  # For provisionVolume: "true":
+  #   A new volume will be created along with a new Ceph user.
+  #   Requires admin credentials (adminID, adminKey).
+  # For provisionVolume: "false":
+  #   It is assumed the volume already exists and the user is expected
+  #   to provide path to that volume (rootPath) and user credentials (userID, userKey).
+  provisionVolume: "true"
+
+  # Ceph pool into which the volume shall be created
+  # Required for provisionVolume: "true"
+  pool: cephfs_data
```

Comment:
this will require a new storage class and provisioner just for rook. Current the CSI external provisioner doesn't resolve the parameters in storageclasses. It merely passes through into the node server and node server is able to get the pool. 

It is surely a better usability to pass a CRD and let provisioner resolve the filesystem (or even create one) and then pass to the node server. I'll open an follow up issue on this.